go/cmd/tool/tool.go

```diff
@@ -0,0 +1,104 @@
+// Binary tool is for quick testing of the queue.
```

Comment:
go/cmd/tool/tool.go path looks too generic? up to you though.

Reply:
Renamed to go/cmd/try-queue/tryqueue.go